hippunfold/config/snakebids.yml

```diff
@@ -278,8 +286,14 @@ template_files:
     crop_refT1w: resources/CITI168/T1w_300umCoronalOblique_hemi-{hemi}.nii.gz
     Mask_crop: resources/CITI168/Mask_300umCoronalOblique_hemi-{hemi}.nii.gz
 
-
-
+atlas: bigbrain
+atlas_files:
+  bigbrain:
+    label_nii: resources/bigbrain/sub-bigbrain_hemi-{hemi}_label-hipp_desc-manualsubfields.nii.gz
```

Comment:
minor bids-naming comment here -- this file is missing a suffix..
suffix should be dseg here, I can make this change when I go through it now